### DISCUSSION TITLE
Stopping Zoom Reset in Plotter

### DISCUSSION THREAD

**Discussion by JeffPS13** [marcomusy/vedo#957] [Q&A] [answered]
Hi Marco,

I have a Plotter instance that I add/change elements in and I repeatedly call a function like: 
`self.plt.show(axes=1,size=(screen_scale*840,screen_scale*595),bg='gray', bg2='bb') ` 

However, every time I add/change it resets the zoom. Is there a way to avoid the zoom reset?

Thanks,
Jeff

**Comment by marcomusy** [comment, marked as answer]
Hi Jeff, just call `.show(...)` once and then inside your callback you just need to call `plt.render()` to refresh.
Check out `examples/basic/mousehover1.py`

**Reply by JeffPS13** [reply]
Thanks, Marco!
